IndicoV2/IndicoClient.cs

```diff
@@ -56,7 +63,38 @@ public IndicoClient(string apiToken, Uri baseUri, bool verify = true)
             _apiToken = apiToken ?? throw new ArgumentNullException(nameof(apiToken));
             BaseUri = baseUri ?? throw new ArgumentNullException(nameof(baseUri));
             _verifySsl = verify;
-            HttpClient = new HttpClient(new AuthenticatingMessageHandler(baseUri, apiToken));
+            var handler = new AuthenticatingMessageHandler(baseUri, apiToken);
```

Comment:
Why did you split this out into two lines here?

Reply:
It was mainly so that we can use the handler in the GraphQLHttpClient as well per line `71`.